lib/html_processor/amp_img.rb

```diff
@@ -0,0 +1,62 @@
+require_dependency 'html_processor/base'
+
+module HTMLProcessor
+  class AmpImg < Base
+    PERMITTED_AMP_ATTRIBUTES = %w[
+      src
+      srcset
+      sizes
+      alt
+      attribution
+      height
+      width
+    ].freeze
+
+    def process(*xpaths)
+      doc.xpath(*xpaths).each do |original_img|
+        if original_img.attributes.include? 'src'
```

Comment:
I think the standard is to use parentheses for parameters, @Tr4pSt3R ?

Reply:
@margOnline yes, that's the case. Parameters ought to have parenthesis.